core/modules/layout/layout.install

```diff
@@ -35,6 +35,21 @@ function layout_install() {
   db_query("UPDATE {system} SET weight = 60 WHERE name = 'layout' AND type = 'module'");
 }
 
+/**
+ * Set home page menu to render as a drop-down menu.
+ */
+function layout_update_1000() {
+  $layout = layout_load('home');
```

Comment:
Let's add a check to see if the layout is still there in the first place. `layout_load()` will return FALSE if the given layout does not exist.